src/encoded/types/file.py

```diff
@@ -1290,20 +1291,20 @@ def download(context, request):
             "cid": "555",
             "an": "4DN Data Portal EC2 Server",     # App name, unsure if used yet
             "ec": "Serverside File Download",       # Event Category
-            "ea": "Range Query" if request.range else "File Download", # Event Action
+            "ea": "Range Query" if request.range else "File Download",  # Event Action
             "el": filename,                         # Event Label
             "ev": file_size_downloaded,             # Event Value
             # Product fields
             "pa": "purchase",
             "ti": str(uuid4()),                     # We need to send a unique transaction id along w. 'transactions' like purchases
             "pr1id": file_at_id,                    # Product ID/SKU
             "pr1nm": filename,                      # Product Name
-            "pr1br" : lab.get("display_title"),     # Product Branch
+            "pr1br": ab.get("display_title"),     # Product Branch
```

Comment:
```suggestion
            "pr1br": lab.get("display_title"),     # Product Branch
```
I have no idea of the context, but sounds like "lab" potentially made sense?